runtime/rastrace/method_trace.c

```diff
@@ -474,13 +474,48 @@ traceMethodArgObject(J9VMThread *thr, UDATA* arg0EA, char* cursor, UDATA length)
 	if (object == NULL) {
 		j9str_printf(PORTLIB, cursor, length, "null");
 	} else {
-		J9Class* clazz = J9OBJECT_CLAZZ(thr, object);
-		J9ROMClass * romClass = clazz->romClass;
-		J9UTF8* className = J9ROMCLASS_CLASSNAME(romClass);
+		/* string arg */
```

Comment:
This comment is misplaced; it belongs before, or (perhaps better) after line 481. There's no need to abbreviate "argument".